Support custom conatiner registry (#4039)
demisto/demisto-sdk: README.md

## XSOAR CI/CD
For information regarding XSOAR CI/CD, please see [this article](https://xsoar.pan.dev/docs/reference/packs/content-management)

## Custom Container Registry

By default, the `demisto-sdk` will use `dockerhub` as the container registry to pull the integrations and scripts docker image.

> It is possible to provide a different container registry. In order to configure it, the following environment variables must be set:

Reviewer: ```suggestion
In order configure a custom container registry, the following environment variables must be set:
```
